test/phoenix/router/helpers_test.exs

```diff
@@ -23,41 +23,43 @@ defmodule Phoenix.Router.HelpersTest do
   defmodule Router do
     use Phoenix.Router
 
-    get "/posts/top", PostController, :top, as: :top
-    get "/posts/bottom/:order/:count", PostController, :bottom, as: :bottom
-    get "/posts/:id", PostController, :show
-    get "/posts/file/*file", PostController, :file
-    get "/posts/skip", PostController, :skip, as: nil
+    scope "/", trailing_slash: true do
```

Comment:
I don't think we should add trailing slash  to all of our routes. Choose a select few (or preferably add a couple of new ones for testing).

Reply:
Agreed. I made this change just to get the tests passing in their current form.  I'm going to revert these changes, as well as the test changes that added the trailing slashes to all the paths.

I'll add a new scope to the test router, and a block of tests to exercise the new option.